bump: :completion vertico (and some requirements)

iyefrat opened the pull request:
---
bump: :completion vertico compat consult-lsp magit git-commit

emacs-straight/compat@7ca7d300d1d2 -> emacs-straight/compat@2bedcb5ea919
mhayashi1120/Emacs-wgrep@f9687c28bbc2 -> mhayashi1120/Emacs-wgrep@edf768732a56
minad/consult-flycheck@7a10be316d72 -> minad/consult-flycheck@51b1b48e8dad
minad/consult@16b2dc5e34c8 -> oantolin/orderless@ae849b3d9f8c
minad/marginalia@c1365bf0c7b5 -> minad/marginalia@6d48ed54be87
minad/vertico@bedd146c3ffc -> minad/vertico@f303790546ed
oantolin/embark@629cce948c56 -> oantolin/embark@4882b395cef9
oantolin/orderless@847694e78c12 -> minad/vertico@f303790546ed
tumashu/vertico-posframe@a3d0802d7b4a -> tumashu/vertico-posframe@790f74b49d53
gagbo/consult-lsp@58b541476203 -> gagbo/consult-lsp@f8db3252c0da
magit/magit@0ef98ef51811 -> magit/magit@6d325d90ba13

consult-lsp needed to be bumped to work with some changed consult
internals, and due to compat shenanigans we have to bump magit to latest
as well.

Includes fixes to stay up to date with upstream api changes to.
consult-customize and the deprecation of embark-define-keymap

---

feat(vertico): remap Info-search to consult-info

---

GTrunSec commented:
really nice. it works well on my channel.

minad commented:
Hey @iyefrat, you may also want to update `+vertico--consult--fd-builder` since I changed the convention for the command builders. I recommend that you check all the Vertico-specific commands added by Doom in the Vertico module one by one if you upgrade. Fortunately there are not that many. My recent Compat work triggered quite a bit of activity in my packages downstream with some breaking changes, but I expect that this will cool down soon.

iyefrat commented:
Thanks @minad! Will do.

iyefrat commented:
Fixed, looked over the rest of the function definitions in the module, nothing else was effected (well, except for things i already addressed in #7023), except for perhaps `+lookup/in-docsets`, which i don't have set up and can't test. I'll try to get to it tomorrow.

hlissner commented:
LGTM! I'll merge this after #7023.

hlissner commented:
And done! Thanks again!
